opentech/apply/funds/templates/funds/base_submissions_table.html

```diff
@@ -8,25 +8,29 @@
 
 {% block content %}
     {% block table %}
+    <div class="wrapper wrapper--table-actions">
         {% if table.data or active_filters %}
-        <div class="button button--filters button--contains-icons js-open-filters">Filter By</div>
+            <div class="button button--filters button--contains-icons js-toggle-filters">Filters</div>
```

Comment:
is there a reason you're going with a `div` and not `button`?

Reply:
Think this was from the first phase of work, but agree that they should all be buttons. Will swap.